lib/img-workers.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const P = require('./promise');
+
+const AppError = require('./error');
+const config = require('./config');
+const logger = require('./logging')('img_workers');
```

Comment:
This file did not exist prior to this PR, so it will likely be changes from `routes.avatar.upload...` to `img_workers.upload`.